### DISCUSSION TITLE
Wagtail Forms Localized?

### DISCUSSION THREAD

**Discussion by enzedonline** [wagtail/wagtail-localize#382] [Q&A] [answered]
Are wagtailforms compatible with localize? 

If so, any documentation/demos on how to do this someone could point me to?

Thanks :)

**Comment by kaedroho** [comment, marked as answer]
Hi @enzedonline. Yes they are! You can translate form pages just like any other page. The main difference is the submissions will be separated by language since each language has it's own page instance

**Reply by enzedonline** [reply]
Hi @kaedroho 
I see the form page (the wagtail page holding the form) with translatable fields as normal, but nowhere to translate the form labels/help text etc. Is this not available via the localize UI?

**Reply by kaedroho** [reply]
Ahh, sorry forgot to mention: You need to add ``TranslatableMixin`` to the form field model and resubmit the page for translation, then they should become translatable

**Reply by enzedonline** [reply]
Ah ... perfect. Thank you.
Should have deleted the page before attempting to migrate ... took far too long to unpick that mess ... be nice if Django migrations were actually transactional rather than leaving you halfway through when things go pear shaped. Live and learn.
